.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Report a bug encountered while freezing you program or runing your frozen program
 title: ''
-labels:
+labels: needs-triage
 assignees: ''
 ---
 
```

Comment:
Could we also turn the sample responses such as `(paste here)` and `e.g Linux/Windows/...` into comments so that it's easier for us (and hopefully them) to see if they've left a field blank.

Reply:
And/or possibly turn the **Context information** into a table?

Reply:
Yes and yes. I'll do this tomorrow.